cfme/web_ui/mixins.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
-from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import fill, Form, AngularSelect, Table, toolbar, form_buttons, flash
 from xml.sax.saxutils import quoteattr
+
+from cfme.fixtures import pytest_selenium as sel
 from cfme.utils.version import current_version
+from cfme.web_ui import fill, Form, AngularSelect, Table, toolbar, form_buttons
```

Comment:
web_ui.Form was removed in #6150

Reply:
mixin is used in one test module, cloud_infra_common.test_tag_objects.

Need to update that test module, and then this mixin can be removed

Reply:
This is in #6154 actually